concordium-sdk/src/main/java/com/concordium/sdk/responses/transactionstatus/PartsPerHundredThousand.java

```diff
@@ -10,15 +10,16 @@
 @Getter
 @ToString
 public class PartsPerHundredThousand {
-    private final Fraction value;
+    private final UInt32 numerator;
 
     private final int HUNDRED_THOUSAND = 100000;
-    PartsPerHundredThousand(UInt64 value) {
-        this.value = new Fraction(value, UInt64.from(HUNDRED_THOUSAND));
+
+    PartsPerHundredThousand(UInt32 value) {
+        this.numerator = value;
     }
 
-    public static PartsPerHundredThousand from(long x) {
-        return new PartsPerHundredThousand(UInt64.from(x));
+    public static PartsPerHundredThousand from(int x) {
+        return new PartsPerHundredThousand(UInt32.from(x));
```

Comment:
Shouldn't this check that x is small enough?

Reply:
It's interpreted as an unsigned 32bit integer, so it can exceed uint 32 max size. Or what do you mean?

Reply:
Should be <= 100_000, no?